readme.md

```diff
@@ -317,6 +317,7 @@ void OnCustomizeElement(GridCustomizeElementEventArgs args)
     }
 }
 ```
+
```

Comment:
Is the comment on line 313 (above) editable? If so, can we change it to:

`/* Add your logic here to check if there are detail rows for the current master row (masterRowKey)*/`